[Core][Channel] Add contact phone number to channel

twojtylak opened the pull request:
| Q               | A
| --------------- | -----
| Branch?         | master 
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | mentioned in #11886
| License         | MIT

Add channel contact phone number

Duplicate of #11943 (Fucked up the rebase process 😭 )

lchrusciel reviewed (commented):
Hey @twojtylak,

thanks for your proposal. It could really land into the core. However, I'm not sure if it is used anywhere. Can you customize the checkout template to make usage of this new field?
![image](https://user-images.githubusercontent.com/6213903/97307216-b73b2b00-185f-11eb-9081-ed13f60d59d0.png)

twojtylak commented:
Hey @lchrusciel,

I adjusted the `_support.html.twig` template. The 'need_assistance' block is now only displayed if the channel has a contact phone number. I also adjusted the fixtures so that something is displayed here after loading them.

lchrusciel commented on src/Sylius/Bundle/CoreBundle/Form/Extension/ChannelTypeExtension.php:
```diff
@@ -75,6 +76,10 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'label' => 'sylius.form.channel.contact_email',
                 'required' => false,
             ])
+            ->add('contactPhoneNumber', TextType::class, [
```
Could be improved to:
```suggestion
            ->add('contactPhoneNumber', TelType::class, [
```
Ref. https://symfony.com/doc/current/reference/forms/types/tel.html

lchrusciel commented:
Thank you, @twojtylak! :tada:
